spec/aruba/api_spec.rb

```diff
@@ -698,7 +698,7 @@
       context 'when file_name contains fixtures "%" string' do
         let(:runtime) { instance_double('Aruba::Runtime') }
         let(:config) { double('Aruba::Config') }
-        let(:environment) { instance_double('Aruba::Environment') }
+        let(:environment) { object_double(Aruba::Environment.new) }
```

Comment:
why `object_double` when we just used `double` on the line above. Let's be consistent.